apps/mail/app/(routes)/settings/general/page.tsx

```
@@ -258,6 +261,43 @@ export default function GeneralPage() {
+                  <FormItem className="bg-popover flex w-full flex-col rounded-lg border p-4">
+                    <div className="space-y-0.5">
+                      <FormLabel className="text-base">
+                        {t('pages.settings.general.trustedSenders')}
+                      </FormLabel>
+                      <FormDescription>
+                        {t('pages.settings.general.trustedSendersDescription')}
+                      </FormDescription>
+                    </div>
+                    <div className="flex flex-col gap-2">
+                      {field.value.map((senderEmail) => (
+                        <div className="flex items-center justify-between">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)